examples/reorder-animation/index.js

```diff
@@ -90,7 +90,11 @@ function render () {
     m.offset = last ? last.offset + last.elmHeight + margin : margin
     return acc.concat(m)
   }, [])
-  totalHeight = data[data.length - 1].offset + data[data.length - 1].elmHeight
+  if (data.length === 0) {
+    totalHeight = 0
+  } else {
+    totalHeight = data[data.length - 1].offset + data[data.length - 1].elmHeight
+  }
```

Comment:
Thank you, @2604150210. Just changing this to a ternary, if you don't mind. Awesome change. Good catch.

```suggestion
  totalHeight = data.length === 0
    ? 0
    : data[data.length - 1].offset + data[data.length - 1].elmHeight
```